lib/chef/provider/package/cab.rb

```diff
@@ -45,7 +45,7 @@ def remove_package(name, version)
         end
 
         def dism_command(command)
-          shellout = Mixlib::ShellOut.new("dism.exe /Online #{command} /NoRestart", { :timeout => @new_resource.timeout })
+          shellout = Mixlib::ShellOut.new("dism.exe /Online /English #{command} /NoRestart", { :timeout => @new_resource.timeout })
```

Comment:
hrm, well this look fine to me then, we need to fix that Mixlib::ShellOut call at some point tho...